grpc-native/src/main/java/org/ballerinalang/net/grpc/nativeimpl/serviceendpoint/FunctionUtils.java

```diff
@@ -151,6 +151,10 @@ private static Object startServerConnector(BObject listener, ServicesRegistry se
     public static Object externStart(BObject listener) {
         ServicesRegistry.Builder servicesRegistryBuilder = getServiceRegistryBuilder(listener);
 
+        if (servicesRegistryBuilder.getServices().isEmpty()) {
+            return null;
```

Comment:
shall we return an error or log message saying like `the listener start is terminated because no attached services found in the listener`

Reply:
or similar message

Reply:
return error will lead to panic. we can log the message

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-grpc/pull/300/commits/af856da6cb8e20c48dee85a12eb9275623e59f83